api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/DataPropertyTemplateModelCallable.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.mannlib.vitro.webapp.web.templatemodels.individual;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import edu.cornell.mannlib.vitro.webapp.beans.DataProperty;
+import edu.cornell.mannlib.vitro.webapp.beans.Individual;
+import edu.cornell.mannlib.vitro.webapp.controller.VitroRequest;
+
+public class DataPropertyTemplateModelCallable implements Callable<DataPropertyTemplateModel> {
```

Comment:
Should we make this Callable<PropertyTemplateModel>, so that the code that process the results can use Future<PropertyTemplateModel> rather than Future<?> ?